spec/requests/admin/search_spec.rb

```diff
@@ -37,6 +37,7 @@
         expect(page).to have_link("Clear")
 
         click_link("Clear")
+        sleep 1
```

Comment:
Was required on CI to keep the test from being flaky; always passes on local either way.